src/components/Counter.js

```diff
@@ -20,10 +26,10 @@ class Counter extends Component {
         return (
             <p>
                 Clicked: {this.props.count} times
-                <button onClick={() => /* Fill me in */ }>
+                <button onClick={() => this.props.increment() }>
                     +
                 </button>
-                <button onClick={() => /* Fill me in */ }>
+                <button onClick={() => this.props.decrement() }>
```

Comment:
Why pass your `onClick`s like this with an extra step? You could just pass the definition of your increment/decrement methods:
```javascript
<button onClick={this.props.increment}>
  // ...
<button onClick={this.props.decrement}>
  // ...
```